src/main/java/uk/gov/hmcts/sscs/service/SubmitAppealService.java

```diff
@@ -87,6 +98,19 @@ private CaseDetails createCaseInCcd(CaseData caseData) {
         try {
             return ccdService.createCase(caseData);
         } catch (CcdException ccdEx) {
+            log.info("Failed to create ccd case but carrying on [" + caseData.getCaseReference() + "]");
+            log.debug("Exception from CCD", ccdEx);
```

Comment:
It will be useful if the exception is also logged at info level. Do we need two log level?